docs_app/src/app/shared/location.service.ts

```diff
@@ -64,28 +64,29 @@ export class LocationService {
   }
 
   search() {
-    const search: { [index: string]: string|undefined; } = {};
+    const search: { [index: string]: string | undefined } = {};
     const path = this.location.path();
     const q = path.indexOf('?');
     if (q > -1) {
       try {
-          const params = path.substr(q + 1).split('&');
-          params.forEach(p => {
-            const pair = p.split('=');
-            if (pair[0]) {
-              search[decodeURIComponent(pair[0])] = pair[1] && decodeURIComponent(pair[1]);
-            }
-          });
-      } catch (e) { /* don't care */ }
+        const params = path.substr(q + 1).split('&');
+        params.forEach((p) => {
+          const pair = p.split('=');
+          if (pair[0]) {
+            search[decodeURIComponent(pair[0])] = pair[1] && decodeURIComponent(pair[1]);
+          }
+        });
+      } catch (e) {
+        /* don't care */
+      }
     }
     return search;
   }
 
-  setSearch(label: string, params: { [key: string]: string|undefined}) {
+  setSearch(label: string, params: { [key: string]: string | undefined }) {
     const search = Object.keys(params).reduce((acc, key) => {
       const value = params[key];
-      return (value === undefined) ? acc :
-        acc += (acc ? '&' : '?') + `${encodeURIComponent(key)}=${encodeURIComponent(value)}`;
+      return value === undefined ? acc : (acc += (acc ? '&' : '?') + `${encodeURIComponent(key)}=${encodeURIComponent(value)}`);
```

Comment:
```suggestion
      return value === undefined ? acc : (acc + (acc ? '&' : '?') + `${encodeURIComponent(key)}=${encodeURIComponent(value)}`);
```
`=` is not needed since you already return the result which will be the new acc

Reply:
Which needs to become new `acc`, therefore it is needed 🙂

Anyways, I didn't change anything in this PR, a lot of fixes here are just Prettier uglifying things (I don't like Prettier), therefore, if it worked before, I wouldn't change it now.